x/staking/client/cli/tx.go

```diff
@@ -594,15 +569,5 @@ func BuildCreateValidatorMsg(clientCtx client.Context, config TxCreateValidatorC
 		return txBldr, msg, err
 	}
 
-	if generateOnly {
```

Comment:
`func BuildCreateValidatorMsg(clientCtx client.Context, config TxCreateValidatorConfig, txBldr tx.Factory, generateOnly bool) (tx.Factory, sdk.Msg, error) {`

`generateOnly` param is necessary?